src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuilding.java

```diff
@@ -839,28 +839,32 @@ public void setBuildingLevel(final int level)
     public final void markDirty()
     {
         dirty = true;
-        if(colony != null)
+        if (colony != null)
         {
             colony.markBuildingsDirty();
         }
     }
 
     /**
-     * register a block and position.
+     * register a TileEntity and position.
      *
-     * @param block to be registered
-     * @param pos of the block
+     * @param world the world the TileEntity is being registered in
+     * @param pos   of the block
      */
-    public void registerBlockPosition(@NotNull Block block, @NotNull final BlockPos pos)
+    public void registerBlockPosition(@NotNull final BlockPos pos, @NotNull final World world)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "block". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1172)